src/main/java/org/motechproject/prevac/repository/VisitDataService.java

```diff
@@ -261,4 +261,46 @@ long countFindByClinicIdVisitTypeAndPlannedVisitDate(
             @LookupField(name = "type") Set<VisitType> typeSet,
             @LookupField(name = "dateProjected") Range<LocalDate> plannedDate);
 
+    @Lookup
+    List<Visit> findByVisitPlannedDateRangeAndVisitTypeSet(
+            @LookupField(name = "dateProjected") Range<LocalDate> plannedDate,
+            @LookupField(name = "type") Set<VisitType> typeSet);
+
+    @Lookup
+    List<Visit> findByVisitPlannedDateRange(
```

Comment:
move it to ui lookups (there we have lookups, that are actually displayed to the user and here lookups changed by lookupHelper)